use-cases.html

```diff
@@ -0,0 +1,287 @@
+            register for location updates. If the user's position changes, the widget
+            can adapt the content accordingly.
+          </p>
+        </section>
+
+        <section id="usecase_poi_alert">
+          <h4>Alerts when points of interest are in the user's vicinity</h4>
+
+          <p>
+            A tour-guide web application can use the Geolocation Sensor API to monitor the
+            user's position and trigger visual or audio notifications when interesting
+            places are in the vicinity. An online task management system can trigger
+            reminders when the user is in the proximity of landmarks that are
+            associated with certain tasks.
+          </p>
+        </section>
+
+        <section id="usecase_show_position">
+          <h4>Show the user's position on a map</h4>
+
+          <p>
+            A user finds themselves in an unfamiliar city area. They want to check their
+            position so they use their handheld device to navigate to a web-based
+            mapping application that can pinpoint their exact location on the city map
+            using the Geolocation Sensor API. They then ask the web application to provide
+            driving directions from their current position to their desired destination.
+          </p>
+        </section>
+
+      </section>
+
+      <section>
+        <h3>Background—Continuous geolocation updates</h3>
+
+        <section id="usecase_turn_by_turn">
+          <h4>Turn-by-turn route navigation</h4>
+
+          <p>
+            A mapping application can help the user navigate along a route by
+            providing detailed turn-by-turn directions. The application does this by
+            registering with the Geolocation Sensor API to receive repeated location updates
+            of the user's position. These updates are delivered as soon as the
+            implementing user agent determines that the position of the user has
+            changed, which allows the application to anticipate any changes of
+            direction that the user might need to do.
+            The application can be in the foreground, but likewise can be backgrounded,
+            for example, when the user turns their device off to save battery
+            on a long highway route section without side roads.
+          </p>
+        </section>
+
+        <section>
+          <h4>Tracking sports activity</h4>
+
+          <p>
+            A web application can track a user's sports activity, for example,
+            a marathon run or a bicycle race. Therefore, the application does not
+            need to be on the screen, but would be backgrounded while the user
+            performs their activity, maybe with their handheld device strapped to their arm.
+          </p>
+        </section>
+
+        <section id="usecase_content_annotation">
+          <h4>Annotating content with location information</h4>
```

Comment:
Why does this use case require background-continuous geolocation updates? It seems similar to the social networking example above except that it uses background sync to upload data when connectivity is available. The geolocation data itself is associated with actions the user took when the application was in the foreground.

Reply:
I have removed this use case altogether. It is covered in one form or the other in the other use cases.